packages/cli/package.json

```diff
@@ -16,9 +16,7 @@
   },
   "scripts": {
     "build": "node scripts/build.js",
-    "prepack": "node scripts/pre-pack.js",
-    "postpack": "node scripts/cleanup.js",
-    "test": "vitest --testTimeout=300000"
```

Comment:
Is it intentional to have the `test` script removed? I think we should keep it to keep testing the tests in the CLI package.

Reply:
This is also coming from https://github.com/stackblitz/tutorialkit/commit/e444b43035711de4128ce8de79ee3ecb6b5d3294#diff-6e2e2a1851648938b325ba84de634407a4e69a644ea61102df15ca4a8a7a9758 🤔 

Looking at the `packages/cli/test`, I think the `<root>/integration` is intended to replace these. But there's test like `cannot create project without installing but with starting` that we should still keep running from here. I'll add `test` script back, `vitest` dependency back and remove those duplicate test cases.

Reply:
I think it might indeed have been the case. However, there's value in keeping them because:
* They tests continuously the CLI with the workspace version
* Integrations are intended to test with the released version

However, I'm not 100% sure what @SamVerschueren had in mind. Sam am I correct? Or did you intend for those to be removed?

Reply:
Ok, let's keep both. 👍